### PR TITLE
Lpfg 828 Attendee list on event overview page

### PR DESCRIPTION
**Pull request by mick-jones**
- Added new _events-overview stylesheet with styles for attendee list
- Added attendee listing on events overview page. 
- Mobile responsiveness added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
 		"log4js": "2.10.0",
 		"moment": "2.22.2",
 		"node-cache": "4.2.0",
+		"number-to-date-month-name": "^1.0.2",
```
Comment (m24murray) [inline package.json]: remove the ^

File: src/applicationContext.ts [hunk 2]
```diff
@@ -217,9 +223,14 @@ export class ApplicationContext {
 		this.dateRangeCommandValidator = new Validator<DateRangeCommand>(this.dateRangeCommandFactory)
 		this.dateRangeFactory = new DateRangeFactory()
 		this.dateRangeValidator = new Validator<DateRange>(this.dateRangeFactory)
+		this.eventRecordFactory = new EventRecordFactory()
+
+		this.learnerRecordConfig = new LearnerRecordConfig('http://localhost:9000')
```
Comment (m24murray) [inline src/applicationContext.ts]: hardcoded var?

File: src/csrs/service/csrsService.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ export class CsrsService {
 	}
 
 	async getOrganisations() {
-		return await this.restService.get('organisationalUnits')
+		return await this.restService.get('organisations')
```
Comment (m24murray) [inline src/csrs/service/csrsService.ts]: I believe this should be `organisationalUnits` as per new csrs changes

File: views/page/course/module/events/events-overview.html [hunk 4]
```diff
@@ -92,7 +91,22 @@ <h3 class="govuk-heading-m">Event overview</h3>
...
+                  {% for attendee in eventRecord %}
+                      {% if attendee.status === 0 %}
+                        {% set payment = "Booked" %}
+                      {% elif attendee.status === 1 %}
+                        {% set payment = "Cancelled" %}
+                      {% elif attendee.status === 2 %} 
+                         {% set payment = "Pending" %}
+                      {% endif %}
+                      <li class="list-details"><p class="govuk-body title">{{ attendee.delegateEmailAddress}}</p><p class="govuk-body payment">{{ payment }}</p><p class="buttons govuk-body"><a class="govuk-link" href="#">View</a></p></li>
+                  {% endfor %}
```
Comment (m24murray) [inline views/page/course/module/events/events-overview.html]: Where should the href be going to here?

File: src/leaner-record/index.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+	private _eventRecordFactory: EventRecordFactory
+
+	constructor(config: LearnerRecordConfig, auth: Auth, eventRecordFactory: EventRecordFactory) {
+		this._restService = new OauthRestService(config, auth)
+
+		this._eventRecordFactory = eventRecordFactory
+	}
+
+	async getEventRecord(eventId: string) {
+		const data = await this._restService.get('/events/' + eventId)
```
Comment (m24murray) [inline src/leaner-record/index.ts]: For consistency sake, could we use string interpolation:

`const data = await this._restService.get(`/events/${eventId}`)`

File: src/lib/datetime.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
+import moment = require('moment')
```
Comment (m24murray) [inline src/lib/datetime.ts]: Are you sure we should have this file? It appears to have been deleted in a previous commit

**Comment by m24murray** [comment]
on hold while we focus on private beta

File: src/applicationContext.ts (hunk 2 above)
Comment (mick-jones) [reply src/applicationContext.ts]: @m24murray back end issue, no idea where this came from maybe @Peter-D-Taylor or @jakub-kainos?

**Comment by m24murray** [comment]
Closing PR as this has been resolved by LPFG-446

**Pull request closed without merging**